src/app/components/profile-location/profile-location.component.ts

```diff
@@ -31,14 +38,45 @@ export class ProfileLocationComponent implements OnInit {
     });
   }
 
-  updatesContactInfo(){
+  getGoogleApiKey()  {
+    this.http.get(`${environment.loginUri}getGoogleApi`)
+       .subscribe(
+                 (response) => {
+                     //console.log(response);
+                     if(response["googleMapAPIKey"] != undefined){
+                         new Promise(() => {
+                         this.userService.googleApiKey = response['googleMapAPIKey'][0];
+                         console.log(`google api key obtained is: ${this.userService.googleApiKey}`);
```

Comment:
Definitely need to remove this one too - we don't want the API key to be logged.